src/server/journalism.js

```diff
@@ -9,6 +11,48 @@ const client =
     });
 
 export const journalismTeam = (res, team) => {
+  return newPromiseChain()
+    .then(() => db.query('SELECT FROM team WHERE name = :name LIMIT 1', { params: { name: team } }))
+    .then((results) => {
+      const searchTeamDbInfo = results[0];
+      return { dbInfo: searchTeamDbInfo };
+    }).then((all) => {
+      return newPromiseChain()
+        .then(() => fetchFromFootballAPI(`http://api.football-data.org/v1/teams/${all.dbInfo.id}/fixtures`))
+        .then((data) => ({ ...all, footballApiData: data }));
+    }).then((all) => (
+      getTeamInformation(team).then((results) => ({ ...all, leftTeam: results }))
+    )).then((all) => {
+      return Promise.all(
+        all.footballApiData.fixtures.slice(0, 10).map(
```

Comment:
Are we not getting all the fixture teams ? X) I guess if they click on one of the fixtures that hasn't got the other team's information, we could make a call to team information in which case you need to have another endpoint similiar to the one you had previously... If you do this method, which 10 are sliced ? that could be 10 last fixtures of the season which are no where near the current date 

otherwise get rid of the slice.